packages/react-next/src/components/FocusTrapZone/examples/FocusTrapZone.Box.Example.tsx

```diff
@@ -8,10 +8,20 @@ import { TextField, ITextFieldStyles } from '@fluentui/react-next/lib/TextField'
 import { Toggle, IToggle } from '@fluentui/react-next/lib/Toggle';
 import { memoizeFunction } from '@fluentui/react-next/lib/Utilities';
 import { useBoolean } from '@uifabric/react-hooks';
+import { getEdgeChromiumNoHighContrastAdjustSelector } from '@fluentui/react-next/lib/Styling';
 
 const getStackStyles = memoizeFunction(
   (useTrapZone: boolean): Partial<IStackStyles> => ({
-    root: { border: `2px solid ${useTrapZone ? '#ababab' : 'transparent'}`, padding: 10 },
+    root: {
+      border: `2px solid ${useTrapZone ? '#ababab' : 'transparent'}`,
+      padding: 10,
+      selectors: {
+        '*': {
+          forcedColorAdjust: 'auto',
+        },
+        ...getEdgeChromiumNoHighContrastAdjustSelector(),
```

Comment:
perhaps the better fix is to set border color to be `Window` here?

Reply:
I think that would be good, I would still have to add a similar line above to fix the styling for the children otherwise they would also adopt the Window color. What do you think?

Reply:
I've set the border color to Window!